### ISSUE TITLE
How to interpret what the worse survival is and what good survival is?

### ISSUE THREAD

**Issue by hyjforesight** [sunduanchen/Scissor#20]
Hello Scissor,
Thanks for developing this amazing package.
In the 'Detecting a hypoxic subpopulation related to worse survival' section of the Results in Nature Biotechnology paper, you input the clinical survival informaton of 471 TCGA-LUAD samples and found that 201 Scissor+ cells were associated with worse survival.

We're wondering how you define what survival is worse survival.如何定义什么是差预后？
What is the boundary between worse survival and good survival, living longer than 5 years as good survival, or the survival range of the top 50% people with the longest survival as good survival?差预后和好预后的界限在哪里？是否生存期>5年的定义为好预后？或者选择生存期最好的50%人群，将其表达矩阵作为好预后的特征矩阵，然后进行Pearson检验和COX回归？
If I manually make a matrix that only includes TCGA samples with very bad survival (like live shorter than 3 months), does Scissor still generate the good survival-related cells?如果我人为构建一个生存期均小于3个月的TCGA样本矩阵，Scissor是否仍将在单细胞数据中找到‘好预后’细胞和‘差预后’细胞？

Thanks!
Best,
YJ

**Comment by sunduanchen** [comment]
Hi YJ,

Thanks for your questions.

There are no manually defined boundary between the worse survival and good survival. The quantiles of survival time (months) for the 471 TCGA-LUAD samples were showed as the following message:

```
        0%        25%        50%        75%       100% 
  0.033333   2.133333   8.200000  25.366667 227.066667
```
For the result interpretation, the meanings of Scissor+ cells (regression coefficient beta > 0) and Scissor- cells (regression coefficient < 0) are consistent with the meanings in the standard cox regression model: **a positive coefficient indicates a worse prognosis and a negative coefficient indicates a protective effect of the variable**. 

In our case, we claimed that the Scissor identified 201 Scissor+ cells were associated with the worse survival (or to say shorter survival time) means that these cells had a 'bad' effect on the patient survival. If a patient has higher expressions of the signature genes of these 201 cells, he/she should have a higher probability with shorter survival time. That is why we investigated the signature genes in our downstream analyses. 

As for your last question, Scissor maybe can identify several Scissor- cells that are linked with good survival. Here 'good survival' is a relative concept when compared with bad survival. In your case, the Scissor- cells are actually associated with the bad survival patients with relatively longer survival times. Notably, probably the difference between good and bad survival in your case is not significant (reliability significance test p-value > 0.05).

Hope these help. 

Best,
Duanchen
